admin_site/system/views.py

```diff
@@ -910,7 +916,8 @@ def form_valid(self, form):
         except OutdatedClientError as e:
             set_notification_cookie(
                 response,
-                _('Computer {0} must be upgraded in order to join a group with scripts attached').format(e),
+                _('Computer {0} must be upgraded in order to join a group with'
+                    'scripts attached').format(e),
```

Comment:
There is a space missing between "with" and "scrips" here, as in the similar cases below.

Reply:
Check